docs/BREAKING_CHANGES.v3.md

```diff
@@ -13,6 +13,11 @@ For more information, see the [KoliBri Maintenance and Support Strategy](https:/
 - Visually, the tooltip has been replaced by a simple label shown in parentheses after the abbreviation.
 - The property `_tooltipAlign` has been removed.
 
+## All Input Components
+
+- The property `_alert` has been removed. It's not being handled automatically based on `_msg` and the touched state. See #6138.
```

Comment:
Es wird doch jetzt automatisch durch _msg und _touched gesetzt, oder?

Reply:
Typo, es sollte `now` statt `not` heißen